Kconfig

```
@@ -68,4 +68,31 @@ config NV_COUNTER_TRACKER_THRESHOLD_LIMIT
 	help
 	  Threshold limit to write current protected storage-based NV counter value to persistent memory.
 
+config NETMON_STACK_SIZE
+	int "Size of stack for network monitor thread"
+	default 4096
+	help
+	  Size of the stack to be used by the netmon worker thread.
+
+config SNTP_INIT_SERVER
```

`INIT` doesn"t really make sense to me here. Maybe just:
```suggestion
config SNTP_SERVER
```

---

Or `SNTP_SERVER_NAME`